gaffer-worker/src/main/java/uk/gov/gchq/gaffer/auth/JwtUserDetailsService.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.auth;
+
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+import java.util.ArrayList;
+
+@Service
+public class JwtUserDetailsService implements UserDetailsService {
+
+    @Override
+    public UserDetails loadUserByUsername(final String username) throws UsernameNotFoundException {
+        if ("javainuse".equals(username)) {
```

Comment:
For now that's okay but we need to properly think about how authentication will work.

Reply:
It's still a todo/work-in-progress, looking to implement a better solution in this ticket #32 in another branch. I guess it currently works for now as a 'dev mode 'whilst gaffer-experimental is still in this experimental development phase, if that's ok?

Can implement a better solution in to this branch also?